index.bs

```diff
@@ -3181,6 +3251,10 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. Set |source|'s [=attribution source/number of event-level reports=] to 0 if
     |source|'s [=attribution source/randomized response=] is null,
     [=attribution source/randomized response=]'s [=list/size=] otherwise.
+1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] obtained with |randomizedResponseConfig|.
+1. If |sourceType| is "<code>[=source type/event=]</code>" and |states| is greater than |source|'s [=attribution source/max event states=]:
+    1. Run [=obtain and deliver debug reports on source registration=] with "<code>[=source debug data type/source-max-event-states-limit=]</code>" and |source|.
```

Comment:
Do we care about reporting the max event states limit when issuing the `source-max-event-states-limit` debug report for an event type source exceeding it? I figure they'll already have it in hand but I'm not sure if we wanted to be consistent with the `source-trigger-state-cardinality-limit` debug report, which [does include the trigger state cardinality limit](https://wicg.github.io/attribution-reporting-api/#ref-for-source-debug-data-type-source-trigger-state-cardinality-limit).

Reply:
Thinking about this some more, I'm not sure that this new debug report type should exist, as it's effectively a form of parsing error resulting in an internal inconsistency between the source registration's actual number of states and what it has chosen to be the maximum number of states. Maybe we can omit it for now and consider re-adding it in the future once we ponder the best design.

Reply:
Agreed that it's effectively a form of parsing error, but we check the limit here to reduce the complexity in parsing due to obtaining number of possible trigger states. I think it would still be beneficial to add this new debug report type so that each source registration would continue to create a verbose debug report.

This limit is determined by both the ad-tech and the user agent's default value, and adding the limit to the report    SGTM.

Reply:
> we check the limit here to reduce the complexity in parsing due to obtaining number of possible trigger states

I don't feel strongly, but regardless of where the check is performed, we have the freedom to report the problem using whatever debug type we want.

Compare this with validation of the `event_level_epsilon` field.

Reply:
Per chat offline, we will use a new debug type. @feifeiji89 can you add the limit to the report?

Reply:
@feifeiji89 Please add the limit to the report.

Reply:
Done.